UVAtlas/isochart/SymmetricMatrix.hpp

```diff
@@ -43,7 +47,11 @@ namespace Isochart
             value_type scale,
             size_t dwDimension)
         {
+#ifndef __LINUX__
```

Comment:
can we not just stud the `std` implementation in both cases?

Reply:
I was hoping to keep the MS path in tact and just patch where needed for linux.

Reply:
fair